RadixWallet/Clients/CloudBackupClient/CloudBackupClient+Live.swift

```diff
@@ -234,14 +228,13 @@ extension CloudBackupClient {
 				}
 			},
 			migrateProfilesFromKeychain: {
-				let activeProfile = await profileStore.profile.id
 				guard let headerList = try secureStorageClient.loadProfileHeaderList() else { return [] }
 
 				let previouslyMigrated = userDefaults.getMigratedKeychainProfiles
 
 				let migratable = try headerList.compactMap { header -> (Data, Profile.Header)? in
 					let id = header.id
-					guard !previouslyMigrated.contains(id), header.id != activeProfile else { return nil }
```

Comment:
This was put in place to avoid uploading empty profiles which where created right when the Wallet was opened. Not needed anymore, as now the new Profile is created and saved only when user goes to the New Wallet user flow.